### DISCUSSION TITLE
Can the jdk17 version be used to deploy Kafka cluster instances?

### DISCUSSION THREAD

**Discussion by peter-hst** [strimzi/strimzi-kafka-operator#7910] [Q&A]
**Is your feature request related to a problem? Please describe.**
We want to try JDK17 to open ZGC and do some performance tests

**Describe the solution you'd like**
Does Strimzi have jdk-related configuration? For example, I can choose OpenJDK17 or zulu-jdk?
We want to try JDK17 to open ZGC and do some performance tests, Thanks a lot!
Can the following options be enabled if the memory space is less than 32G?
-XX:+UseCompressedClassPointers
-XX:+UseCompressedOops

**Comment by scholzj** [comment]
Java 17 will be used from Strimzi 0.33 release. For older releases, you can probably use it as runtime, but you need to do your own custom build.

**Reply by peter-hst** [reply]
Thank a lot Scholzj,  The version we use is 0.32, it works well with ingress
